contracts/DAIInterestRateModelV4.sol

```diff
@@ -0,0 +1,132 @@
+pragma solidity ^0.8.10;
+
+import "./JumpRateModelV2.sol";
+
+/**
+  * @title Compound's DAIInterestRateModel Contract (version 4)
+  * @author Compound, Dharma (modified by Maker Growth)
+  * @notice The change from v3 to v4 of this contract was just the `SECONDS_PER_BLOCK` constant,
+  * as noted in https://github.com/compound-finance/compound-protocol/issues/230
+  */
+contract DAIInterestRateModelV4 is JumpRateModelV2 {
+    uint256 private constant BASE = 1e18;
+    uint256 private constant RAY_BASE = 1e27;
+    uint256 private constant RAY_TO_BASE_SCALE = 1e9;
+    uint256 private constant SECONDS_PER_BLOCK = 12;
+
+    /**
+     * @notice The additional margin per block separating the base borrow rate from the roof.
+     */
+    uint public gapPerBlock;
+
+    /**
+     * @notice The assumed (1 - reserve factor) used to calculate the minimum borrow rate (reserve factor = 0.05)
+     */
+    uint public constant assumedOneMinusReserveFactorMantissa = 0.95e18;
+
+    PotLike pot;
+    JugLike jug;
+
+    /**
+     * @notice Construct an interest rate model
+     * @param jumpMultiplierPerYear The multiplierPerBlock after hitting a specified utilization point
+     * @param kink_ The utilization point at which the jump multiplier is applied
+     * @param pot_ The address of the Dai pot (where DSR is earned)
+     * @param jug_ The address of the Dai jug (where SF is kept)
+     * @param owner_ The address of the owner, i.e. the Timelock contract (which has the ability to update parameters directly)
+     */
+    constructor(uint jumpMultiplierPerYear, uint kink_, address pot_, address jug_, address owner_) JumpRateModelV2(0, 0, jumpMultiplierPerYear, kink_, owner_) public {
+        gapPerBlock = 4e16 / blocksPerYear;
```

Comment:
I think that the gap should be discussed. 

ETH-B stability fee + 4% = 7%, vs 5.13% at the moment.

Reply:
I don't think I get this. Should we discuss this in the governance forum?

Reply:
I meant that there is no real reason why we would keep the same as before. @monnet-supply proposed some parameters here, for example: https://www.comp.xyz/t/compound-dsr-proposal/3856/7?u=mathisgd